vnext/Chakra/ChakraJsiRuntime.h

```diff
@@ -0,0 +1,241 @@
+#pragma once
+
+#if defined(USE_EDGEMODE_JSRT)
+#include <jsrt.h>
+#else
+#include <ChakraCore.h>
+#endif
+
+#include "ChakraJsiRuntimeArgs.h"
+
+#include <jsi/jsi.h>
+
+#include <memory>
+#include <mutex> 
+#include <sstream>
+#include "UnicodeConversion.h"
+
+namespace facebook { 
+namespace jsi { 
+namespace chakraruntime {
+
+class ByteArrayBuffer : public jsi::Buffer {

```

Comment:
>ByteArrayBuffer [](start = 6, length = 15)

Looking at the previously abandoned internal CR, we will be using this class in reatnativehost independently from ChakraJsiRuntime. I feel that we should move this class into its own header.

Reply:
Yichen.. I'll take it as part of another change.